endpoints/bahmni/input-validation.json

```diff
@@ -1,553 +1,203 @@
-        },
-        "postalCode": {
-          "type": "string"
-        },
-        "countyDistrict": {
-          "type": "string"
-        },
-        "address3": {
-          "type": "string"
-        },
-        "address4": {
-          "type": "string"
-        },
-        "address5": {
-          "type": "string"
-        },
-        "address6": {
-          "type": "string"
-        },
-        "startDate": {
-          "type": "timestamp"
-        },
-        "endDate": {
-          "type": "timestamp"
-        },
-        "latitude": {
-          "type": "number"
-        },
-        "longitude": {
-          "type": "number"
-        },
-        "voided": {
-          "type": "boolean",
-          "default": false
-        },
-        "address7": {
-          "type": "string"
-        },
-        "address8": {
-          "type": "string"
-        },
-        "address9": {
-          "type": "string"
-        },
-        "address10": {
-          "type": "string"
-        },
-        "address11": {
-          "type": "string"
-        },
-        "address12": {
-          "type": "string"
-        },
-        "address13": {
-          "type": "string"
-        },
-        "address14": {
-          "type": "string"
-        },
-        "address15": {
-          "type": "string"
-        },
-        "links": {
-          "type": "array",
-          "element": {
-            "rel": {
-              "type": "string"
-            },
-            "uri": {
-              "type": "string"
+          },
+          "links": {
+            "type": "array",
+            "items": {
+              "type": "object",
+              "properties": {
+                "uri": {
+                  "type": "string"
+                },
+                "rel": {
+                  "type": "string"
+                }
+              }
             }
           }
-        },
-        "resourceVersion": {
-          "type": "string",
-          "required": true
         }
       }
     },
-    "attributes": {
-      "type": "array",
-      "element": {
-        "display": {
-          "type": "string"
-        },
-        "uuid": {
-          "type": "string"
-        },
-        "value": {
-          "type": "string"
-        },
-        "attributeType": {
-          "element": {
-            "uuid": {
-              "type": "string"
-            },
-            "display": {
-              "type": "string"
-            },
-            "links": {
-              "type": "object",
-              "rel": {
+    "person": {
+      "type": "object",
+      "properties": {
+        "names": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "display": {
                 "type": "string"
               },
-              "uri": {
+              "familyName": {
+                "type": "string"
+              },
+              "middleName": {
+                "type": "string"
```

Comment:
```suggestion
                "type": "string",
                "nullable": true
```

Reply:
fixed in https://github.com/jembi/openhim-mediator-mapping/pull/21/commits/da90c901b05f8fcd2480d722da8ccaa7f6a86d77